testutil/random.go

```diff
@@ -87,7 +87,15 @@ func RandomValidator(t *testing.T) *eth2v1.Validator {
 
 func RandomAttestation() *eth2p0.Attestation {
 	return &eth2p0.Attestation{
-		AggregationBits: RandomBitList(),
+		AggregationBits: RandomBitList(1),
+		Data:            RandomAttestationData(),
+		Signature:       RandomEth2Signature(),
+	}
+}
+
+func RandomAggregateAttestation() *eth2p0.Attestation {
+	return &eth2p0.Attestation{
+		AggregationBits: RandomBitList(64),
```

Comment:
According to spec: length of AggregationBits should be `MAX_VALIDATORS_PER_COMMITTEE` which is 2048. Setting this 64 might be problematic when testing with VCs on simnet if they check this on their end.

Reply:
lets cross that bridge when/if we get there